packages/firebase/compat/index.d.ts

```diff
@@ -5215,6 +5215,8 @@ declare namespace firebase.analytics {
 
     /**
      * Use gtag 'config' command to set 'screen_name'.
+     *
+     * @deprecated Use logEvent with eventName as 'screen_view' and add relevant eventParams.
```

Comment:
Same as above. Unfortunately links work a little differently in this file as you have to specify the namespace and you're allowed to customize the link text I guess. Search for "@link" in this file for examples, one example is {@link firebase.analytics.Analytics `Analytics`} Again, generate docs and make sure the link looks right.

Reply:
Added the `{@links logEvent}` and added backticks where necessary. The `logEvent` link here auto resolves to the compat version `firebase.analytics.Analytics.logEvent` in vscode and in the resulting html files post docgen:compat